backend/src/schema/types/type/NOTIFIED.gql

```diff
@@ -0,0 +1,28 @@
+type NOTIFIED {
+  from: NotificationSource
+  to: User
+  createdAt: String
+  read: Boolean
+  reason: NotificationReason
+}
+
+union NotificationSource = Post | Comment
+
+enum NotificationOrdering {
+  createdAt_asc
+  createdAt_desc
+}
+
+enum NotificationReason {
+  mentioned_in_post
+  mentioned_in_comment
+  commented_on_post
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-30T08:32:49Z" title="Friday, August 30th 2019, 10:32:49 am +02:00">Aug 30, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,28 @@
+type NOTIFIED {
+  from: NotificationSource
+  to: User
+  createdAt: String
+  read: Boolean
+  reason: NotificationReason
+}
+
+union NotificationSource = Post | Comment
+
+enum NotificationOrdering {
+  createdAt_asc
+  createdAt_desc
+}
+
+enum NotificationReason {
+  mentioned_in_post
+  mentioned_in_comment
+  comment_on_post
```


I know that I did wrote this, but shouldn’t that be `commented_on_post`?
To have the database plausible. This we can not just change very easy after the release …

(This change would have a little effect on the frontend as well …)